drivers/modem/gsm_ppp.c

```diff
@@ -1124,13 +1145,62 @@ static void gsm_configure(struct k_work *work)
 	gsm_ppp_unlock(gsm);
 }
 
+static int pin_init(void)
+{
+	int rc = 0;
+
+#if DT_INST_NODE_HAS_PROP(0, mdm_wdisable_gpios)
+	rc = gpio_pin_configure_dt(&wdisable_gpio, GPIO_OUTPUT_LOW);
+	if (rc < 0) {
+		LOG_ERR("Failed to configure wdisable pin");
+		return rc;
+	}
+	gpio_pin_set_dt(&wdisable_gpio, 0);
+#endif
+
+#if DT_INST_NODE_HAS_PROP(0, mdm_dtr_gpios)
+	gpio_pin_set_dt(&dtr_gpio, 0);
+	rc = gpio_pin_configure_dt(&dtr_gpio, GPIO_OUTPUT_LOW);
```

Comment:
you should likely use `_INACTIVE`, this is using dt flags (where you can have active low/high)